### PR TITLE
HomePage added

### PR DESCRIPTION
**Pull request by Jayvashishtha123**
Homepage created.....

https://github.com/HackSlashNITP/LibraLink/assets/132293279/ca8e7f94-efef-47db-ade7-35abb3c1f65d

Screenshot_1698597712
Screenshot_1698597766

Screenshot_1698597745

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -10,7 +12,7 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return const MaterialApp(
-      home: LandingPage(),
+      home: HomePage(),
```
Comment (RitwikSingh28) [inline lib/main.dart:23]: Change the `home` property to `LandingPage` to show it by default

**Review by RitwikSingh28 — CHANGES_REQUESTED** [review]
@Jayvashishtha123 
Splendid work. The page looks great and neat.
Just make one little change and we will be okay to merge your work

**Review by ShubhamTiwari55 — COMMENTED** [review]
@Jayvashishtha123 Solve your merge conflicts and then inform me so that I can merge your PR

**Review by ShubhamTiwari55 — APPROVED** [review]
Good work @Jayvashishtha123 Merging this PR

**Pull request merged**